playbooks/roles/foreman_libvirt/tasks/libvirt.yml

```diff
@@ -1,4 +1,9 @@
 ---
```

Comment:
It's pretty specific to Foreman, especially with how the networking is setup.  The virsh module is already pretty good, anyway, if users want to write their own role to setup libvirt in the way they'd like.